images/kubectl-build-deploy-dind/helmcharts/k8up-schedule/templates/schedule.yaml

```diff
@@ -19,6 +19,7 @@ spec:
     schedule: '{{ .Values.check.schedule }}'
   prune:
     retention:
-      keepDaily: 7
-      keepWeekly: 6
+      keepDaily: '{{ .Values.prune.retention.keepDaily }}'
```

Comment:
for each value used in helm templates we should also define a default in the helmchart `values.yaml`, can you do that for these new variables?